src/messages/layout.js

```diff
@@ -47,6 +47,19 @@ export const TYPE_NO_MANIFEST_JSON = {
     will be attempted to be inferred by package layout.`),
 };
 
+export const TYPE_MULTIPLE_FILES = {
```

Comment:
That said if this matches the previous validator I guess we might be better to keep it like this for easier comparisons.